doc/source/toolchain.rst

```diff
@@ -271,9 +271,9 @@ making and distributing a SciPy release.
 References
 ----------
 
-.. [1] https://docs.scipy.org/doc/scipy/reference/release.1.2.0.html
+.. [1] :doc:`release.1.2.0`
 .. [2] https://python3statement.org
-.. [3] https://docs.scipy.org/doc/numpy/release.html
+.. [3] :doc:`release`
```

Comment:
Where is this supposed to go? It used to go to:
[1.17 Release Notes](https://docs.scipy.org/doc/numpy/release.html)
Now it goes to the [list of SciPy Release Notes](https://16426-1460385-gh.circle-artifacts.com/0/html-scipyorg/release.html).

Reply:
Whoops!  Looks like I have a bad assumption in my script that I'll need to fix.